base/record_replay.cc

```diff
@@ -205,7 +211,21 @@ void* IdPointer(int id) {
   return OP2(V8RecordReplayIdPointer(id), nullptr);
 }
 
-AutoLockMaybeEventsDisallowed::AutoLockMaybeEventsDisallowed(base::Lock& lock) : lock_(lock) {
+void OnMouseEvent(const char* kind,
+                                size_t clientX,
+                                size_t clientY) {
+  OP(V8RecordReplayOnMouseEvent(kind, clientX, clientY));
+}
+void OnKeyEvent(const char* kind, const char* key) {
+  OP(V8RecordReplayOnKeyEvent(kind, key));
+}
+void OnNavigationEvent(const char* kind, const char* url) {
+  OP(V8RecordReplayOnNavigationEvent(kind, url));
+}
+
+AutoLockMaybeEventsDisallowed::AutoLockMaybeEventsDisallowed(
```

Comment:
I don't understand how this snuck in here... Investigating now

Reply:
looks like there was some auto-formatting?

Reply:
Indeed!